oci_spec/src/caps_migrations.rs

```diff
@@ -0,0 +1,92 @@
+use caps::Capability;
+use crate::LinuxCapabilityType;
+
+// utility function to switch between the types LinuxCapabilityType and Capabality
+pub fn to_caps(local_cap: &LinuxCapabilityType) -> Capability {
```

Comment:
It is better to use the `Into` trait.
https://doc.rust-lang.org/std/convert/trait.Into.html

Reply:
I agree but we would need to be able to impl the `From` trait for `caps::Capability` which cannot be done with an external crate. 

See idea below to fix this :)

Reply:
I must of been sleepy when I looked at it last night you can totally `impl From` for an external crate! updated :+1: